templates/layouts/one-column-overlay.html.twig

```diff
@@ -0,0 +1,22 @@
+{% if 'centered-container' in settings.centered|render and region_attributes.main is iterable %}
+  {% set region_attributes_main = region_attributes.main.addClass('flex-12-of-12') %}
+  {% set region_attributes = region_attributes|merge({'main': region_attributes_main}) %}
+{% endif %}
+
+{% if layout_builder_admin %}
+  <div{{ attributes.addClass('jumpstart-ui--one-column', settings.centered, settings.extra_classes) }}>
+    <div {{ region_attributes.main.addClass('main-region') }}>
+      <div>{{ 'Background Image'|t }}</div>
+      {{ content.main }}
+    </div>
+
+    <div {{ region_attributes.overlay.addClass('overlay-region') }}>
+      <div>{{ 'Overlaying Contents'|t }}</div>
+      {{ content.overlay }}
+    </div>
+  </div>
+{% else %}
+  {% set hero_body = content.overlay %}
+  {% set hero_image = content.main %}
+  {% include "@decanter/components/hero/hero.twig" %}
```

Comment:
{% set data = {hero_body: content.overlay, hero_image: content.main, attributes: attributes} %}
{% include "@decanter/components/hero/hero.twig" with data only %}

Reply:
ahh i see. so that the other variables in this template aren't passed down. gotcha :+1:

Reply:
Yeah, there have been some unintentional consequences with leaving it open to global inheritance in a few instances but overall it hasn't been bad.